datalad/distributed/ora_remote.py

```diff
@@ -486,7 +492,17 @@ def rename(self, src, dst):
         self._run('mv {} {}'.format(sh_quote(str(src)), sh_quote(str(dst))))
 
     def remove(self, path):
-        self._run('rm {}'.format(sh_quote(str(path))))
+        try:
+            self._run('rm {}'.format(sh_quote(str(path))))
+        except RemoteCommandFailedError as e:
+            if "permission denied" in str(e).lower():
+                raise RIARemoteError(
+                    str(e) + os.linesep +
+                    "No support for fixing your permissions. Note,"
+                    " that write permissions to the directory are "
+                    "required to drop remote content.")
+            else:
+                raise
```

Comment:
In addition to the wording change (see above), I would prefer to remove the conditional. The exception is given in both cases anyways and the additional not is just that. The chances to miss it because of an `rm` that reports slightly different are bigger than the confusion, IMHO.